index.html

```diff
@@ -57,6 +160,13 @@
                 // {
                 //     "Short text": { "type": "text" } // default name is 'Single-line free text'; change it here
                 // },
+                {
+                    "Custom Element": {
+                        "inputTag": "validated-input",
+                        "attributes": {
+                        }
+                    }
+                },
                 {
                     "Email": {
```

Comment:
form builder

<img width="1624" alt="1-form-builder" src="https://github.com/user-attachments/assets/ff4d7b51-977b-44d9-844f-991957f9e152">
<img width="1624" alt="2-form-builder" src="https://github.com/user-attachments/assets/211ba7be-adb2-48ab-99be-2188d5e9e412">

collect data

<img width="1624" alt="3-collect-data" src="https://github.com/user-attachments/assets/10b8ee11-598b-4f7b-a454-6cdfb2a30fcd">

![4-collect-data](https://github.com/user-attachments/assets/c9c1f482-04b0-402d-bde5-1ec055b15b95)


validation

![5-validation-required](https://github.com/user-attachments/assets/7948ac3f-0a26-46cb-89d2-65f30dedc251)

![6-validation-custom](https://github.com/user-attachments/assets/da7a422e-8a25-4ebc-a0a5-de5c9558bc1c)

Reply:
lol the `input` doesn’t have `name` attribute 🤦‍♂️ will fix

Reply:
> lol the input doesn’t have name attribute 🤦‍♂️ will fix

fixed in 179ff9f

![Screenshot 2024-09-14 at 8 56 08 PM](https://github.com/user-attachments/assets/84f82fed-a7e4-418e-a73a-3b537aa6cd6d)

![Screenshot 2024-09-14 at 8 56 21 PM](https://github.com/user-attachments/assets/2687e79e-3f95-40e7-ac64-ba759ae59265)

Reply:
> fixed in https://github.com/choonkeat/tiny-form-fields/commit/179ff9fe5af8a9c3aad91f2f66910683b3d89170

think it was our web component js. reverted the attribute change in [74d2dc1](https://github.com/choonkeat/tiny-form-fields/pull/7/commits/74d2dc12540bb20fb816fd5ff1d37bd39b0f5487#diff-2dd82f159d96fbfcd26fb7d885d25e0d54efde9e19a42494b416fa84a5aca568)